projects/divvy-bet/index.js

```diff
@@ -1,24 +1,47 @@
 const { Program } = require("@project-serum/anchor");
-const { getProvider, sumTokens2, } = require("../helper/solana");
-const { PublicKey, SystemProgram } = require("@solana/web3.js");
+const { PublicKey } = require("@solana/web3.js");
+const { getProvider, sumTokens2 } = require("../helper/solana");
 
 async function tvl() {
+  function findHouseAuthorityAddress(houseKey) {
+    return PublicKey.findProgramAddressSync(
+      [Buffer.from("House Authority"), houseKey.toBuffer()],
+      DIVVY_PROGRAM_ID
+    )[0];
+  }
 
-  const programId = new PublicKey('dvyFwAPniptQNb1ey4eM12L8iLHrzdiDsPPDndd6xAR')
-  const provider = getProvider()
-  const idl = await Program.fetchIdl(programId, provider)
-  const program = new Program(idl, programId, provider)
-  const houses = await program.account.house.all()
-  const tokensAndOwners = houses.map(house => {
-    const owner = PublicKey.findProgramAddressSync([house.publicKey.toBuffer(), Buffer.from(house.account.authorityBump.toString())], SystemProgram.programId)[0]
-    return [house.account.currency, owner]
-  })
-  return sumTokens2({ tokensAndOwners })
+  const NATIVE_MINT = "So11111111111111111111111111111111111111112";
+  const DIVVY_ADMIN = "AHf1MX99d31ebLfAydVPe2vVdgzZGuUaW972znWPNzZY";
+
+  const programId = new PublicKey("dvyFwAPniptQNb1ey4eM12L8iLHrzdiDsPPDndd6xAR");
+  const provider = getProvider();
+  const idl = await Program.fetchIdl(programId, provider);
+  const program = new Program(idl, programId, provider);
+  const houses = await program.account.house.all([
+    {
+      memcmp: {
+        offset: 8,
+        bytes: DIVVY_ADMIN, // houses governed by Divvy's multisig
+      },
+    },
+  ]);
+  const tokensAndOwners = houses
+    .filter((house) => house.account.currency.toBase58() !== NATIVE_MINT)
+    .map((house) => {
+      const owner = findHouseAuthorityAddress(house.publicKey, programId);
+      return [house.account.currency, owner];
+    });
+  const solOwners = houses
```

Comment:
@g1nt0ki FYI - this should be tracked as solOwner because it stores native SOL, not SPL wrapped SOL. Thanks!